src/analysis.zig

```diff
@@ -3482,6 +3482,7 @@ pub fn getPositionContext(
                     .enum_literal => curr_ctx.ctx = .{
                         .enum_literal = tokenLocAppend(curr_ctx.ctx.loc().?, tok),
                     },
+                    .parens_expr => curr_ctx.ctx = .{ .var_access = tok.loc },
```

Comment:
There already exists a switch case for this:
```zig
.empty, .pre_label, .var_access => curr_ctx.ctx = .{ .var_access = tok.loc },
```
You can add `parens_expr` there instead of duplicating it.